manager/controllers/app/fybrik_notebook_read_flow_test.go

```diff
@@ -47,7 +47,7 @@ func TestS3NotebookReadFlow(t *testing.T) {
 			s != "charts/fybrik/notebook-test-readflow.tls-system-cacerts.yaml") {
 		t.Skip("Only executed for notebook tests")
 	}
-	catalogedAsset, ok := os.LookupEnv("CATALOGED_ASSET")
+	catalogedAsset, ok := os.LookupEnv("OPENMETADATA_CATALOGED_ASSET")
```

Comment:
why do we need a separate env variable - `OPENMETADATA_CATALOGED_ASSET` ? why it cannot be just `CATALOGED_ASSET` ?

Reply:
Fixed. Thanks

Reply:
This PR closes https://github.com/fybrik/fybrik/issues/1772